### ISSUE TITLE
Anamorphic Renders are squished

### ISSUE THREAD

**Issue by pmjdebruijn** [jliljebl/flowblade#727]
Flowblade 2.2 (MLT 6.16)

Import 16:9 footage (in my case 3840x2160)
Place footage on the timeline
Go to render pick any anamorphic format, like "DV/DVD Widescreen PAL" or "HDV 1440x1080p 25 fps"
Look at the rendered footage, it's squished and letterboxed

**Comment by jliljebl** [comment]
I'll give this a look towards end of the cycle.

**Comment by jliljebl** [comment]
Yes, confirmed. If material and render profile have different pixel aspect ratios we are not getting correct results.

This could take a while. Non-square pixel ratios are on their way out and this could end up going into Known Issues category because this is clearly on fringe use case.

**Comment by pmjdebruijn** [comment]
That's an understandable position to take... And indeed use-cases for exporting anamorphic are few and far in between, DVDs being the big exception, but DVDs aren't the hot new thing either :)

Though I do think if this issue isn't getting resolved anytime soon, that you should consider hiding all non-square pixel-aspect-ratio profiles from the export dialog... or possibly even better, hide all pixel-aspect-ratio profiles that are not equal to the project native aspect ratio, this will prevent flowblade from wasting the users time, while doing the first export and having to find out the output isn't usable...

**Comment by jliljebl** [comment]
We have too many Profiles displayed to the user right now, so we need to hide most of by default, and all non-square pixel profiles are certainly going into the hidden group.

Deciding on the default Profiles set and the GUI design + current code structures issues are complex enough to push this to 2.6 but we will quite likely get something there.
